doc/git-subrepo.swim

```diff
@@ -599,6 +599,74 @@ If you want to chat about the `git-subrepo` command, join `#gitcommands` on
 * Written in (very modern) Bash, with full test suite. Take a look.
 * A `.gitrepo` file never is in the top level dir (next to a `.git/` dir).
 
+= Working with nested subrepos
+
+`git-subrepo` supports adding subrepos which themselves make use of
+subrepos. In fact, subrepo itself is built this way. Take a look in the
+`ext/` folder, it houses 2 dependencies as subrepos, `test-more-bash` and
+`bashplus`. 
+
+`test-more-bash` itself depends on 2 more subrepos: `bashplus` again, and
+`test-tap-bash`.
+
+The structure is therefore as follows:
+
+  git-subrepo
+    |- bashplus
+    |- test-more-bash
+      |- bashplus
+      |- test-tap-bash
+
+However, it's important to understand how this works. Here's the key idea:
+*`git-subrepo` is 100% agnostic about your dependencies!*
```

Comment:
I feel for some, especially those that aren't English speakers as a primary language may have issues understanding what 100% agnostic means and what is being referred to by dependency. I feel that the subsequent paragraph describes what is happening enough that the paragraph starting on line 620 is superfluous, and could be safely removed.